apm-agent-plugins/apm-quartz/apm-quartz-common/pom.xml

```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
-        <artifactId>apm-quartz-job-plugin</artifactId>
+        <artifactId>apm-quartz</artifactId>
```

Comment:
[for reviewer] using `apm-quartz` as module name for all the other sub-modules, the `-plugin` sufix is kept for the instrumentation plugins (with an extra per-version suffix to fit the naming conventions).